src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -224,7 +224,7 @@ public Task<CompletionItem> Handle(CompletionItem completionItem, CancellationTo
             var completionList = new CompletionList(completionItems, isIncomplete: false);
 
             // We wrap the pre-existing completion list with an optimized completion list to better control serialization/deserialization
-            CompletionList optimizedCompletionList = null;
+            CompletionList optimizedCompletionList;
```

Comment:
Let the analyzer flow through you.

Reply:
lol whaa?

Reply:
It just looked like an analyzer might have suggested that you remove an unnecessary assignment.

Reply:
oo ya totally did!